lib/resources/google_apis.dart

```diff
@@ -0,0 +1,85 @@
+import 'dart:io' show Platform;
+import 'package:acs_upb_mobile/pages/timetable/model/events/uni_event.dart';
+import 'package:flutter/foundation.dart' show kIsWeb;
+import 'package:googleapis_auth/auth_io.dart';
+import 'package:googleapis/calendar/v3.dart';
+
+class GoogleApiHelper {
+  GoogleApiHelper();
+
+  static const List<String> _scopes = [CalendarApi.CalendarScope];
+
+  static List<String> get scopes => _scopes;
+
+  static ClientId get credentials {
```

Comment:
Can you explain why this is necessary in a comment?

Reply:
It is some info I need for this method:
```
Future<AutoRefreshingAuthClient> clientViaUserConsent(
    ClientId clientId, List<String> scopes, PromptUserForConsent userPrompt,
    {Client baseClient}
```

This is the function for getting the user's consent to export events to Google Calendar. It requires clientID credentials (a client ID is used to identify a single app to Google's OAuth servers, so it is dependent on the app, not on the user). This information is copied from Google Cloud Console.

I also need some scopes. Right now, I chose calendarScope, which allows us to _see, edit, share, and permanently delete all the calendars you can access using Google Calendar_. In case I change my mind,  I might replace this with some other scopes.

Reply if I should resolve this issue.

Reply:
Added a summary of this in code.